src/main/java/Chatbot/alain/Events.java

```diff
@@ -0,0 +1,42 @@
+package chatbot.alain;
+
+/**
+ * Represents a task that is an event occurring within a specific time range.
+ */
+public class Events extends Task {
```

Comment:
Similar to Deadlines, singular task should be in its Singular term, thus the naming would be better off as "Event" class